### ISSUE TITLE
Dual Discriminator Generative Adversarial Network for Video Anomaly Detection

### ISSUE THREAD

**Issue by llien30** [mn1204/paper_summary#105]
# INFO

## Author
Fei Dong, Yu Zhang, And Ziushan nie
## Conference or Year
2020
## Link
- [arXiv]()
- [official GitHub]()

# Abstract
2つのdiscriminatorを持つ、半教師あり学習を利用した、異常検知のための敵対的学習のネットワークを提案
1. Generatorが未来のフレームを予測
2. frame discriminatorとmotion discriminatorがgeneratorと敵対して、よりリアルで連続的なフレームを作成
frame discriminator : フレームが本物か偽物かを判別
motion discriminator : オプティカルフローが本物か偽物かを判別

# Proposed Method
Generatorは時刻t-T 〜 t-1までのフレームから、時刻tでのフレームを作成．(実験時はT=4)
時刻t-1の本物のフレームと偽物の時刻tのフレームからオプティカルフローを算出
* movement loss = 予測したオプティカルフローと本物のオプティカルフローのロス
* frame loss = 予測したフレームと本物のフレームとのロス
<img width="640" alt="Screen Shot 2021-02-10 at 23 13 14" src="https://user-images.githubusercontent.com/51707667/107521162-8f48bb00-6bf5-11eb-8099-971d7e0b3d16.png">
<img width="509" alt="Screen Shot 2021-02-10 at 23 13 31" src="https://user-images.githubusercontent.com/51707667/107521187-98398c80-6bf5-11eb-8d50-0543a2705c98.png">

### テスト時
予測したフレームと本物のフレームとのPSNRを利用して異常度を計算

# Evaluation
<img width="548" alt="Screen Shot 2021-02-10 at 23 17 56" src="https://user-images.githubusercontent.com/51707667/107521696-36c5ed80-6bf6-11eb-82ab-d0bf7d56d802.png">

# Contribution

# Discussion, Future Work

## Date
2021/02/10
